pkg/policy/api/rule.go

```diff
@@ -57,12 +57,28 @@ type Rule struct {
 	// +kubebuilder:validation:Optional
 	Ingress []IngressRule `json:"ingress,omitempty"`
 
+	// IngressDeny is a list of IngressRule which are enforced at ingress.
+	// Any rule inserted here will by denied regardless of the allowed ingress
+	// rules in the 'ingress' field.
+	// If omitted or empty, this rule does not apply at ingress.
+	//
+	// +optional
```

Comment:
Nit: 
```suggestion
	// +kubebuilder:validation:Optional
```

For the sake of consistency.